install_script.sh.template

```diff
@@ -1747,6 +1748,14 @@ function manage_client_libraries_security_config(){
     $sudo_cmd sed -i -n -e '/^DD_APPSEC_SCA_ENABLED=/!p' -e "\$aDD_APPSEC_SCA_ENABLED=$sca_enable" "$etc_environment"
   fi
 }
+function manage_client_libraries_profiling_config(){
+  local sudo_cmd="$1"
+  local etc_environment="$2"
+  local profiling_enable="$3"
+  if [ -n "$profiling_enable" ]; then
+    $sudo_cmd sed -i -n -e '/^DD_PROFILING_ENABLED=/!p' -e "\$aDD_PROFILING_ENABLED=$profiling_enable" "$etc_environment"
```

Comment:
do we want to override existing values ?

Reply:
I'd tend to say yes, currently if the user runs the install script twice, once with `DD_PROFILING_ENABLED` and once  without, they'd keep the first invocation config as far as profiling goes, while I'd expect it to be disabled following the 2nd install

Reply:
Exactly.

Reply:
Oh wait, I probably misunderstood this. You'd expect the _absence_ of the env var in the installer invocation to _explicitly_ disable profiling? I could even accept this to be the expected semantics (although it'd warrant a wider discussion), but then it'd work differently than how the currently already publicly supported DD_APPSEC_ENABLED and DD_IAST_ENABLED vars work. I think all three should be treated identically by the script.